packages/@o3r/schematics/src/utility/ast.ts

```diff
@@ -169,7 +169,7 @@ export const generateBlockStatementsFromString = (str: string) => {
   const sourceFile = ts.createSourceFile(
     'index.ts',
     `function func() {
-      ${str}
+      ${str.replace(/^\s*/g, '')}
```

Comment:
`.trim()` would do the job as well :)

Reply:
`trim` won't remove the spaces at the beginning of a line